types/bulma-calendar/index.d.ts

```diff
@@ -0,0 +1,472 @@
+     *
+     * @default 'datetime'
+     */
+    type?: 'date' | 'time' | 'datetime';
+
+    /**
+     * Picker dominant color
+     *
+     * @default 'primary'
+     */
+    color?: string;
+
+    /**
+     * Range capability (start and end date/time selection
+     *
+     * @default false
+     */
+    isRange?: boolean;
+
+    /**
+     * Possibility to select same date as start and end date in range mode
+     *
+     * @default true
+     */
+    allowSameDayRange?: boolean;
+
+    /**
+     * Display lang (from language supported by date-fns)
+     *
+     * @default navigator.language.substring(0, 2) || "en"
+     */
+    lang?: string;
+
+    /**
+     * Date format pattern
+     *
+     * @default 'MM/DD/YYYY'
+     */
+    dateFormat?: string;
+
+    /**
+     * Time format pattern
+     *
+     * @default 'HH:mm'
+     */
+    timeFormat?: string;
+
+    /**
+     * Display mode
+     *
+     * @default 'default'
+     */
+    displayMode?: 'default' | 'dialog' | 'inline';
+
+    /**
+     * @default 'auto'
+     */
+    position?: string;
+
+    /**
+     * Show/Hide header block (with current selection)
+     *
+     * @default true
+     */
+    showHeader?: boolean;
+
+    /**
+     * Header block position
+     *
+     * @default 'top'
+     */
+    headerPosition?: 'top' | 'bottom';
+
+    /**
+     * Show/Hide footer block
+     *
+     * @default true
+     */
+    showFooter?: boolean;
+
+    /**
+     * Show/Hide buttons
+     *
+     * @default true
+     */
+    showButtons?: boolean;
+
+    /**
+     * Show/Hide Today Button
+     *
+     * @default true
+     */
+    showTodayButton?: boolean;
+
+    /**
+     * Show/Hide Clear Button
+     *
+     * @default true
+     */
+    showClearButton?: boolean;
+
+    /**
+     * Cancel button label
+     *
+     * @default 'Cancel'
+     */
+    cancelLabel?: string;
+
+    /**
+     * Clear button label
+     *
+     * @default 'Clear'
+     */
+    clearLabel?: string;
+
+    /**
+     * Today button label
+     *
+     * @default 'Today'
+     */
+    todayLabel?: string;
+
+    /**
+     * Now button label
+     *
+     * @default 'Now'
+     */
+    nowLabel?: string;
+
+    /**
+     * Validate button label
+     *
+     * @default 'Validate'
+     */
+    validateLabel?: string;
+
+    /**
+     * Enable/disable month switch
+     *
+     * @default true
+     */
+    enableMonthSwitch?: boolean;
+
+    /**
+     * Enable/disable year switch
+     *
+     * @default true
+     */
+    enableYearSwitch?: boolean;
+
+    /**
+     * Pre-selected start date
+     */
+    startDate?: Date;
+
+    /**
+     * Pre-selected end date
+     */
+    endDate?: Date;
+
+    /**
+     * Minimum date allowed
+     */
+    minDate?: Date;
+
+    /**
+     * Maximum date allowed
+     */
+    maxDate?: Date;
+
+    /**
+     * List of disabled dates
+     */
+    disabledDates?: any[];
+
+    /**
+     * List of disabled week days
+     */
+    disabledWeekDays?: any[];
```

Comment:
this looks it should support `string` as well:
https://github.com/Wikiki/bulma-calendar/blob/297506683763e7d5d3becaceec8c3d866c39faa5/src/js/datePicker/index.js#L494